draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1737,27 +1739,26 @@ hash function approximating a random oracle. Farashahi et al. {{FFSTV13}}
 and Tibouchi and Kim {{TK17}} refine this analysis. In particular, Farashahi
 et al. show that summing two independent evaluations of any of the
 deterministic endings of {{encodings}} suffices to approximate a random
-oracle to an elliptic curve. In other words:
-
-~~~
-   hash2curve(alpha) = f(H0(alpha)) + f(H1(alpha))
-~~~
-
-where f: F -> E is a deterministic encoding, and H0 and H1 are hash
-functions (modeled as random oracles) mapping strings to elements of F.
+oracle to an elliptic curve.
 
 ## Implementation
 
-To instantiate a random oracle from the encodings of {{encodings}},
+To instantiate a random oracle from any of the encodings of {{encodings}},
 compute the following:
 
 ~~~
- hash2curve(alpha) = map2curve( alpha || I2OSP(0, 1) )
-                   + map2curve( alpha || I2OSP(1, 1) )
+u0 = hash2base(alpha, 0)
+u1 = hash2base(alpha, 1)
+(x, y) = map2curve(u0) (+) map2curve(u1)
```

Comment:
(+)  -> +  
(remove parenthesis of plus)

Reply:
I think it might be clearer to avoid overloading the `+` operator for point addition, since `+` appears in a lot of the pseudocode and has a different meaning there.

I'm happy to remove it, but it seems like it might improve clarity to use a different symbol here.

Reply:
> map2curve(u0) (+) map2curve(u1)
> where map2curve is the chosen encoding and the (+) operation is elliptic curve point addition.

The note following the equation alerts the reader about performing point addition. So I will go for using a simple plus symbol.

Reply:
That makes sense, and it's true that the reader *should* be able to figure it out. On the other hand, it seems like using different notation draws attention to the fact that this isn't normal addition, which might make it easier on the reader.

I just checked, and it looks like RFC8032 (EdDSA) uses normal `+`. So maybe it makes sense to do the same.

Before I change it back, @chris-wood, @grittygrease, @jedisct1, any further thoughts?

Reply:
I think simply `+` is fine, though maybe in the future we should consider separate notation to separate point from integer arithmetic.